source/includes/v3/_products.md

```diff
@@ -1629,6 +1629,115 @@ woocommerce.get("products").parsed_response
 	<code>tag</code>, <code>shipping_class</code> and <code>pa_*</code> filters are available starting from WooCommerce 2.5.
 </aside>
 
+## Duplicate product ##
```

Comment:
This documentation is for the *legacy* WC API. As such, we shouldn't be updating the doc for this, as our change is for the non-legacy WP REST API.

Reply:
Done in the commit [b8345d7](https://github.com/woocommerce/woocommerce-rest-api-docs/pull/244/commits/b8345d7d191191b038a087404aadea51d30a2799)

Reply:
@mattsherman Could you take another look at this PR?